modules/ROOT/attachments/dist/xslt/XLSX CodeLists to Genericode/README.txt

```diff
@@ -3,9 +3,9 @@ README
 
 Steps to generate the GC files out from the ESPD-EDM Code Lists excel file:
 
-1. Open the file "https://github.com/ESPD/ESPD-EDM/blob/3.0.0/docs/src/main/asciidoc/dist/cl/xlsx/ESPD-CodeLists-V3.0.0.xlsx"
-2. Save it locally as "ESPD-CodeLists-V3.0.0.xml" (Select the following filetype: XML Spreadsheet)
-3. Use your preferred XLS-T transformation method and the Stylesheet "Excel_to_GC_Multilang_v1.0.xsl" as an input to generate the *.gc files (each tab in "ESPD-CodeLists-V02.00.01.xlsx" will be transformed into a separate *.gc file)
+1. Open the file "https://github.com/ESPD/ESPD-EDM/tree/3.0.1/ESPD-EDM/__ESPDTeam__/ESPD-CodeLists.xlsx"
+2. Save it locally as "ESPD-CodeLists.xml" (Select the following filetype: XML Spreadsheet)
+3. Use your preferred XLS-T transformation method and the Stylesheet "excel-to-gc-multilang-v1.0.xsl" as an input to generate the *.gc files (each tab in "ESPD-CodeLists.xlsx" will be transformed into a separate *.gc file)
```

Comment:
Comment for the commit below - image [modules/ROOT/images/Distribution_package.jpg](https://github.com/OP-TED/espd-docs/pull/1/commits/1cb651802963332db7fcd5b0620871499f33bc4f#diff-9d6a815335d012509ee2ab516ec166e72f53de0ece1b8251a4272115184872d3)

Te added image has 'docs' between 'data-model' and java-library', is that correct?

Reply:
Fixed image removing docs folder.